src/test/java/it/gov/pagopa/payment/controller/payment/BasePaymentControllerIntegrationTest.java

```diff
@@ -481,7 +481,7 @@ protected void configureMocks() {
             TransactionCreationRequest trxRequest = TransactionCreationRequestFaker.mockInstance(i);
             trxRequest.setInitiativeId(INITIATIVEID);
 
-            extractResponse(createTrx(trxRequest, "DUMMYMERCHANTID", ACQUIRERID, IDTRXISSUER), HttpStatus.FORBIDDEN, null);
+            extractResponse(createTrx(trxRequest, "DUMMYMERCHANTID", ACQUIRERID, IDTRXISSUER), HttpStatus.NOT_FOUND, null);
```

Comment:
i'm not expecting that this task should change the previous responses! why should this change from FORBIDDEN to NOT_FOUND?

Reply:
where is traced this HttpStatus change?